tests/integration/testsuites/rate-limit/testsuite.go

```diff
@@ -0,0 +1,111 @@
+package ratelimit
+
+import (
+	_ "embed"
+	"log"
+	"path"
+
+	"github.com/cucumber/godog"
+	"k8s.io/client-go/dynamic"
+
+	"github.com/kyma-project/api-gateway/tests/integration/pkg/global"
+	"github.com/kyma-project/api-gateway/tests/integration/pkg/helpers"
+	"github.com/kyma-project/api-gateway/tests/integration/pkg/resource"
+	"github.com/kyma-project/api-gateway/tests/integration/pkg/testcontext"
+)
+
+const manifestsDirectory = "testsuites/rate-limit/manifests/"
+
+func (t *testsuite) createScenario() *scenario {
+	testId := helpers.GenerateRandomTestId()
```

Comment:
When you declare variable with shorthand in it the shorthand should be uppercase.
```suggestion
	testID := helpers.GenerateRandomTestID()
```